lib/ansible/modules/cloud/misc/helm_cli.py

```diff
@@ -0,0 +1,595 @@
+      - absolute URL
+      - Required when release_state is set to present
+    required: false
+    type: str
+  chart_repo_url:
+    description:
+      - Chart repository url where to locate the requested chart
+    required: false
+    type: str
+  chart_repo_username:
+    description:
+      - Chart repository username where to locate the requested chart
+      - Required if chart_repo_password is specified
+    required: false
+    type: str
+  chart_repo_password:
+    description:
+      - Chart repository password where to locate the requested chart
+      - Required if chart_repo_username is specified
+    required: false
+    type: str
+  chart_version:
+    description:
+      - Chart version to install. If this is not specified, the latest version is installed
+    required: false
+    type: str
+  release_name:
+    description:
+      - Release name to manage
+    required: true
+    type: str
+    aliases: [ name ]
+  release_namespace:
+    description:
+      - Kubernetes namespace where the chart should be installed
+      - Can't be changed with helm 2
+    default: "default"
+    required: false
+    type: str
+    aliases: [ namespace ]
+  release_state:
+    choices: ['present', 'absent']
+    description:
+      - Desirated state of release
+    required: false
+    default: present
+    aliases: [ state ]
+    type: str
+  release_values:
+    description:
+        - Value to pass to chart
+    required: false
+    default: {}
+    aliases: [ values ]
+    type: dict
+  tiller_host:
+    description:
+      - Address of Tiller
+      - Ignored when is helm 3
+    type: str
+  tiller_namespace:
+    description:
+      - Namespace of Tiller
+      - Ignored when is helm 3
+    default: "kube-system"
+    type: str
+  update_repo_cache:
+    description:
+      - Run `helm repo update` before the operation. Can be run as part of the package installation or as a separate step.
+    default: "false"
+    type: bool
+
+#Helm options
+  disable_hook:
+    description:
+      - Helm option to disable hook on install/upgrade/delete
+    default: False
+    type: bool
+  force:
+    description:
+      - Helm option to force reinstall, ignore on new install
+    default: False
+    type: bool
+  kube_context:
+    description:
+      - Helm option to specify which kubeconfig context to use
+    type: str
+  kubeconfig_path:
+    description:
+      - Helm option to specify kubeconfig path to use
+    type: path
+    aliases: [ kubeconfig ]
+  purge:
+    description:
+      - Remove the release from the store and make its name free for later use
+    default: True
+    type: bool
+  wait:
+    description:
+      - Wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment are in a ready state before marking the release as successful
+    default: False
+    type: bool
+  wait_timeout:
+    description:
+      - Timeout when wait option is enabled (helm2 is a number of seconds, helm3 is a duration
```

Comment:
```suggestion
      - Timeout when wait option is enabled (helm2 is a number of seconds, helm3 is a duration)
```
Is it clear what this means to a user of the module?